docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.7'
+
+services:
+  postgres:
+    image: postgres:alpine
+    env_file: .env
+    ports: 
+      - "5432:5432"
+
+  redis:
```

Comment:
As I mentioned on the previous comment if you don't use Redis don't configure it.